ledfx/devices/artnet.py

```diff
@@ -41,15 +46,39 @@ class ArtNetDevice(NetworkedDevice):
                 default="",
             ): str,
             vol.Optional(
-                "device_repeat",
-                description="numer of pixels to consume per device and repeat pre and post ambles, 0 default will use all pixels in one instance",
+                "pixels_per_device",
+                description="Number of pixels to consume per device. Pre and post ambles are repeated per device. By default (0) all pixels will be used by one instance",
                 default=0,
             ): vol.All(int, vol.Range(min=0)),
+            vol.Optional(
+                "dmx_start_address",
+                description="The start address within the universe",
+                default=1,
+            ): vol.All(int, vol.Range(min=1)),
             vol.Optional(
                 "even_packet_size",
                 description="Whether to use even packet size",
                 default=True,
             ): bool,
+            vol.Optional(
```

Comment:
Speculative comment, would it be better to just have a drop down with the four exclusive options instead of the bool and the dropdown, so we could have

RGB
RGBW None
RGBW Accurate
RGBW Brighter

Reply:
Yeah I also thought about this option already.